src/utils/constants.js

```diff
@@ -0,0 +1,10 @@
+const Variant = Object.freeze({
```

Comment:
should this file be called `variant.js` ([`airbnb/javascript#23.6`](https://github.com/airbnb/javascript#naming--filename-matches-export)) or are there plans to add other constants to this file?

Reply:
There are no active plans from my part, so this could probably be renamed